homeassistant/components/co2signal/strings.json

```diff
@@ -6,7 +6,7 @@
           "location": "[%key:common::config_flow::data::location%]",
           "api_key": "[%key:common::config_flow::data::access_token%]"
         },
-        "description": "Visit https://electricitymaps.com/free-tier to request a token."
+        "description": "Visit https://www.electricitymaps.com/pricing to request a free api token."
```

Comment:
If this needs to be changed, we should also move the URL into a placeholder. The URL itself should no longer be part of the translation itself.